addons/mail/static/src/components/emojis_list/emojis_list.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.EmojisList" owl="1">
+        <div class="o_EmojisList" t-attf-class="{{ className }}" t-ref="root">
```

Comment:
Actually I thought this was a "split" from emoji popover, but it really just is a rename, so I would advice to split the PR into 2 parts: 1st part just the rename, which we can merge immediately, and then second part the usage of new popover stuff.

Reply:
PR for just renaming emojis popover to emojis list: https://github.com/odoo/odoo/pull/81905